Video Processing/gpu_video_encryptor/video_processor.py

```
@@ -0,0 +1,19 @@
+
+def extract_frames(video_path):
+    cap = cv2.VideoCapture(video_path)
+    frames = []
+    while True:
+        ret, frame = cap.read()
+        if not ret: break
+        frames.append(frame)
+    cap.release()
+    return frames
+
+def save_video(frames, path, fps=30):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)